### PR TITLE
chore(snackbar): message of the snackbar as html content

### PR DESCRIPTION
**Pull request by mladenplaninicic**
Closes #

{{short description}}

## Acceptance Criteria

- [x] Design Review by @Gagne87, @clastzoo
- [x] Technical Review by @hirsch88, @mladenplaninicic, @nobilo
- [x] Visual test done by @hirsch88, @nobilo, @mladenplaninicic, @Gagne87, @clastzoo
- [x] Functional test done by @hirsch88, @nobilo, @mladenplaninicic
- Browser Testing by @hirsch88, @nobilo, @mladenplaninicic, @Gagne87, @clastzoo
    - Desktop
        - [x] Chrome
        - [x] Edge
        - [x] Safari
        - [x] Firefox
    - Tablet
        - [x] iPad (Landscape / Portrait)
    - Mobile
        - [x] Safari iOS
        - [x] Chrome Preview

### CODE REVIEW

File: packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx [hunk 1]
```diff
@@ -131,9 +136,10 @@ export class Snackbar {
               </bal-heading>
             </span>
           </div>
-          <bal-text class="bal-snackbar__label" space={this.action === '' ? 'bottom' : 'none'} innerHTML={this.message}>
+          <span class="bal-snackbar__label" {...labelAttributes}>
             <slot />
```
Comment (nobilo) [inline packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx:140]: i think we need to use named slot in order for mangolia?
also will this not influence as well the inner html with this class?

Comment (madaluca) [reply packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx:140]: I tested with the beta version and it's working in Magnolia. So we don't need the named slot anymore

**Review by madaluca — APPROVED** [review]

File: packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx (hunk 1 above)
Comment (mladenplaninicic) [reply packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx:140]: perfect

Comment (nobilo) [reply packages/components/src/components/notice/bal-snackbar/bal-snackbar.tsx:140]: great, tnx

**Pull request merged**